src/Microsoft.ComponentDetection.Orchestrator/Experiments/DetectorExperiments.cs

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.ComponentDetection.Orchestrator.Experiments;
 /// </summary>
 public static class DetectorExperiments
 {
+    public const bool AutomaticallyProcessExperiments = false;
```

Comment:
shouldnt this be true?

Reply:
no

Reply:
Your logic is:

```csharp
if (!shouldCheckAutomaticProcessFlag && !DetectorExperiments.AutomaticallyProcessExperiments)
{
    return;
}
```
In the `DetectorProcessingService`, `shouldCheckAutomaticProcessFlag` is set to `false`.
`AutomaticallyProcessExperiments` is `false`.

`!shouldCheckAutomaticProcessFlag && !DetectorExperiments.AutomaticallyProcessExperiments`
`!false && !false`
`true && true`
`true` => `FinishAsync` is a No-OP in `DetectorProcessingService`.

It should be true.